custom/dhis2/tasks.py

```diff
@@ -288,64 +214,60 @@ def gen_children_only_ours(domain):
             yield CommCareCase.wrap(doc)
 
 
-@periodic_task(run_every=crontab(minute=4, hour=4))  # Run daily at 04h04
-def sync_child_entities():
+# TODO: Use case forwarding, or form forwarding of registration forms
+@periodic_task(run_every=timedelta(minutes=5))  # Run every 5 minutes to match forwarded forms
+def sync_cases():
```

Comment:
Thanks @czue. I'll get onto that now. There are two ways to improve performance here; using case/form forwarding is one, and using a single query to get the enabled domains is another. The query will be an easy win. I'll do that first.